python/bosdyn-mock/src/bosdyn/mock/services/estop.py

```
@@ -0,0 +1,56 @@
+
+    def EstopCheckIn(self, request, context):  # pylint: disable=unused-argument
+        response = estop_pb2.EstopCheckInResponse()
+        response.header.CopyFrom(build_response_header(request))
+        status, challenge = self._state.estop.check_in(request)
+        response.status = status
+        response.request.CopyFrom(request)
+        if challenge is not None:
+            response.challenge = challenge
+        return response
+
+    def GetEstopConfig(self, request, context):  # pylint: disable=unused-argument
```

Ruff (0.14.4)

Unused method argument: `context`

(ARG002)